src/NServiceBus.Core/Encryption/RijndaelEncryptionService.cs

```diff
@@ -227,6 +227,11 @@ protected virtual bool TryGetKeyIdentifierHeader(out string keyIdentifier, IInco
             return context.Headers.TryGetValue(Headers.RijndaelKeyIdentifier, out keyIdentifier);
         }
 
+        protected virtual void GenerateIV(RijndaelManaged rijndael)
```

Comment:
ah did not give enough attention to the code to see that `RjindalManaged` is a scoped variable and not a field. Rename the method to `ConfigureIV` then and keep the code as is?